easybuild/easyconfigs/a/AMDSMI/AMDSMI-6.4.1-LLVMtc-ROCm-6.4.1.eb

```
@@ -0,0 +1,39 @@
+easyblock = 'CMakeMake'
+
+name = 'AMDSMI'
+version = '6.4.1'
+
+homepage = 'https://github.com/ROCm/amdsmi'
+description = """The AMD System Management Interface (AMD SMI) library offers a unified tool
+for managing and monitoring GPUs, particularly in high-performance computing environments.
+It provides a user-space interface that allows applications to control GPU operations,
+monitor performance, and retrieve information about the system's drivers and GPUs."""
+docurls = ['https://rocmdocs.amd.com/projects/amdsmi/']
+
+toolchain = {'name': 'LLVMtc', 'version': 'ROCm-6.4.1'}
```

```suggestion
toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
```

There should be nothing that requires building `amdsmi` with ROCm-LLVM, see 

https://github.com/easybuilders/easybuild-easyconfigs/blob/856e35188b9ec014cc8f95e13044e14e5d7463f9/easybuild/easyconfigs/a/amdsmi/amdsmi-25.4.2-GCCcore-14.2.0-ROCm-6.4.1.eb

---

Also note the `esmi` components, which are required (building may fail otherwise) and _should not_ be pulled in during the build. We can provide them ourselves.